internal/cmd/config/profile/set/set.go

```diff
@@ -49,6 +50,14 @@ func NewCmd(p *print.Printer) *cobra.Command {
 				return fmt.Errorf("set profile: %w", err)
 			}
 
+			flow, err := auth.GetAuthFlow()
+			if err != nil {
+				p.Debug(print.WarningLevel, "both keyring and text file storage failed to find a valid authentication flow for the active profile")
+				p.Warn("Failed to find a valid authentication flow for the active profile. Please login using the 'stackit auth login' command.\n")
```

Comment:
Make this message more user friendly, "authentication flow" is not something that is user-facing.
It can be just "The active profile "profile-name" is not authenticated, please login ...."